src/views/Posts/Posts.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box } from '@mui/system';
+import React, { useEffect } from 'react';
+import { useLocation, useParams } from 'react-router';
+import Header from '../../components/Header/Header';
+import PostCard from '../../components/PostCard/PostCard';
+import { getPostsList } from '../../redux/actions/posts';
+import { useAppDispatch, useAppSelector } from '../../redux/hooks';
+
+const Posts = () => {
+  const params = useParams();
+  const { state } = useLocation();
+  const { info, isProcessing } = useAppSelector((state) => state.postReducer);
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    if (params.id) {
+      dispatch(getPostsList(params.id));
+    }
+  }, []);
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        padding: 2,
+      }}
+    >
+      <Header
+        list={[
+          {
+            label: 'Posts',
+            to: `/users/${params.id}/posts`,
+            active: true,
+          },
+          {
+            label: 'Todo',
+            to: `/users/${params.id}/todos`,
+          },
+        ]}
```

Comment:
Move the list outside the component. We don't need to re-render it multiple times.

Reply:
done